Makefile

```diff
@@ -131,30 +133,30 @@ check-ancient-pip: $(PY273)
 
 check-pytest: check-pytest28 check-pytest30
 
-check-faker070: $(TOX) $(PY35)
+check-faker070: $(TOX) $(BEST_PY3)
 	$(TOX) -e faker070
 
-check-faker071: $(TOX) $(PY35)
+check-faker071: $(TOX) $(BEST_PY3)
 	$(TOX) -e faker071
 
-check-django18: $(TOX) $(PY35)
+check-django18: $(TOX) $(BEST_PY3)
 	$(TOX) -e django18
 
-check-django110: $(TOX) $(PY35)
+check-django110: $(TOX) $(BEST_PY3)
 	$(TOX) -e django110
 
-check-django111: $(TOX) $(PY35)
+check-django111: $(TOX) $(BEST_PY3)
 	$(TOX) -e django111
 
 check-django: check-django18 check-django110 check-django111
 
 check-examples2: $(TOX) $(PY27)
 	$(TOX) -e examples2
 
-check-examples3: $(TOX) $(PY35)
+check-examples3: $(TOX) $(BEST_PY3)
 	$(TOX) -e examples3
 
-check-coverage: $(TOX) $(PY35)
+check-coverage: $(TOX) $(BEST_PY3)
```

Comment:
Since `$(TOX)` on line 174 gives us `$(BEST_PY3)`, do we actually need a version here? Or would it be enough to do:

```make
check-coverage: $(TOX)
```